packages/syntax-core/src/Typography/Typography.stories.tsx

```diff
@@ -0,0 +1,143 @@
+import { StoryObj, Meta } from "@storybook/react";
+import Typography from "./Typography";
+
+export default {
+  title: "Typography",
+  component: Typography,
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/p7LKna9JMU0JEkcKamzs53/%F0%9F%93%90-Syntax?node-id=4003-11352&t=t9iBh7rbKNhWCMrt-0",
+    },
+  },
+  argTypes: {
+    align: {
+      options: ["start", "center", "end", "forceLeft", "forceRight"],
```

Comment:
What do `forceLeft` and `forceRight` do? What will they do when a language is rtl?

Reply:
@jliotta see the description in on the `align` prop:

>   Aligns the text to the left, right, or center of the container.
>   * `start` and `end` will align the text to the left or right of the container depending on the locale.
>   * `forceLeft` and `forceRight` will align the text to the left or right of the container regardless of the locale (should be used sparingly).

We basically want to encourage folks to use `start` and `end` and only use `forceLeft` / `forceRight` in edge cases